src/app/sagas/effects/callAndRetry.ts

```diff
@@ -36,7 +35,7 @@ export const createCallAndRetry = (options: Partial<Options>) =>
         if ('onError' in o && o.onError) {
           yield call(o.onError, e, attemptNumber);
         }
-        if (attemptNumber < o.maximumAttempts) {
+        if (!o.maximumAttempts || attemptNumber < o.maximumAttempts) {
```

Comment:
What us o referring to? Maybe we could be more explicit with the naming?

Reply:
You’re right, I’ve renamed it to `options`